enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/helper/Suspendable.java

```diff
@@ -19,7 +19,7 @@
  */
 package org.neo4j.causalclustering.helper;
 
-public interface Enableable
+public interface Suspendable
 {
     void enable() throws Throwable;
```

Comment:
`resume` and `suspend` might be better names now that the interface has been renamed.